docs/source/howto/plugin_codes.rst

```diff
@@ -335,39 +366,51 @@ With the entry points set up, you are ready to launch your first calculation wit
         $ verdi computer setup -L localhost -H localhost -T local -S direct -w `echo $PWD/work` -n
         $ verdi computer configure local localhost --safe-interval 5 -n
 
+ *  Create the input files for our calculation
+
+    .. code-block:: console
+
+        $ echo "File with content\ncontent1" > file1.txt
+        $ echo "File with content\ncontent2" > file2.txt
+        $ mkdir input_files
+        $ mv file1.txt file2.txt input_files
+
  * Write a ``launch.py`` script:
 
     .. code-block:: python
 
         from aiida import orm, engine
         from aiida.common.exceptions import NotExistent
-
-        # Setting up inputs
+        from os import path
```

Comment:
DataFactory missing?

Maybe better to also add this as a separate "snippet" to get the benefit of linting

Reply:
```suggestion
        import pathlib
```